test/elixir_analyzer/test_suite/captains_log_test.exs

```diff
@@ -1,93 +1,28 @@
-defmodule ElixirAnalyzer.ExerciseTest.CaptainsLogTest do
```

Comment:
[oops, I though that was added to my review but I guess not]
This looks like a mistake...

Reply:
I have no idea how that change got here 😱